src/sources/forus-webshop/scss/_common/components/_buttons.scss

```diff
@@ -5,7 +5,7 @@
     border: 1px solid transparent;
     padding: 12px 25px;
     transition: color .4s, background .4s, box-shadow .4s, border .4s;
-    border-radius: 6px;
+    border-radius: $border-radius;
```

Comment:
Dont just use a variable in common without checkin if this will influence other implentations. For example, this will result in no border radius for WK implementation while it currenlty had.